MinecraftClient/ChatBots/ChatLog.cs

```diff
@@ -92,6 +100,14 @@ public override void GetText(string text)
             }
         }
 
+        public override void OnInternalCommand(string commandName,string commandParams, string Result)
+        {
+            if (saveInternal)
+            {
+                save($"Internal {commandName}({commandParams}): {Result}");
```

Comment:
String Interpolation is only available starting with C# 6. MCC is targeting .NET 4 with C# 4.
You can easily circumvent this by using String formatting instead:
```C#
save(String.Format("Internal {0}({1}): {2}", commandName, commandParams, result));
```